### PR TITLE
Scene clipping distance implementation

### PR DESCRIPTION
**Pull request by nCastle1**
Moves to 100.7 Runtime and adds support for scene clipping distance.

### CODE REVIEW

File: src/ARToolkit.Forms/ARSceneView.cs [hunk 1]
```diff
@@ -36,6 +36,25 @@ public ARSceneView()
             OriginCamera = new Mapping.Camera(0, 0, 15E6, 0, 0, 0);
         }
 
+        /// <summary>
+        /// Identifies the <see cref="ClippingDistance"/> bindable property.
+        /// </summary>
+        public static readonly BindableProperty ClippingDistanceProperty =
+            BindableProperty.Create(nameof(ClippingDistance), typeof(double), typeof(ARSceneView), 0.0, BindingMode.OneWay, null);
```
Comment (dotMorten) [inline src/ARToolkit.Forms/ARSceneView.cs]: Confirming: Is the default 0.0 and not NaN or something else?

File: src/Directory.Build.props [hunk 2]
```diff
@@ -35,7 +35,7 @@
     <UWPTargetFramework>uap10.0.16299</UWPTargetFramework>
     <NetCoreTargetFramework>netcoreapp3.0</NetCoreTargetFramework>
 
-    <ArcGISRuntimePackageVersion Condition="'$(ArcGISRuntimePackageVersion)'==''">100.6.0</ArcGISRuntimePackageVersion>
+    <ArcGISRuntimePackageVersion Condition="'$(ArcGISRuntimePackageVersion)'==''">100.7.0</ArcGISRuntimePackageVersion>
```
Comment (fragmer) [inline src/Directory.Build.props]: Nice catch!

File: src/ARToolkit.Forms/ARSceneView.cs (hunk 1 above)
Comment (nCastle1) [reply src/ARToolkit.Forms/ARSceneView.cs]: Default is 0, and zero should mean no clipping

**Comment by nCastle1** [comment]
Thanks for the reviews!

**Pull request merged**